src/tools/AddNewRoutePathTool.ts

```diff
@@ -0,0 +1,42 @@
+import RoutePathStore from '~/stores/routePathStore';
+import NotificationStore from '~/stores/notificationStore';
+import NotificationType from '~/enums/notificationType';
+import NodeType from '~/enums/nodeType';
+import RoutePathLinkService from '~/services/routePathLinkService';
+import ToolbarTool from '~/enums/toolbarTool';
+import BaseTool from './BaseTool';
+
+/**
+ * Tool for creating new routePath
+ */
+export default class AddNewRoutePathTool implements BaseTool {
+    public toolType = ToolbarTool.AddNewRoutePath;
+    public activate() {}
+    public deactivate() {}
+
+    public isNetworkNodesInteractive() {
+        const hasRoutePathLinks =
+        RoutePathStore!.routePath &&
+        RoutePathStore!.routePath!.routePathLinks!.length === 0;
+
+        return RoutePathStore!.isCreating && hasRoutePathLinks;
+    }
+
+    public onNetworkNodeClick = async (clickEvent: any) => {
```

Comment:
Do not use any

Reply:
Couldn't find types for these click events (this is how they were before). I tried different types but couldnt get one working.

Found only this: https://github.com/mhasbie/react-leaflet-vectorgrid
(that isnt mentioning type...)

But I added type for this:
gridLayer.on('click', (event: L.LeafletEvent) => {

Reply:
ok